ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -60,3 +60,106 @@ or other node types with `underlay_router: false`.
 - `bgp_peer_groups.IPv4_UNDERLAY_PEERS` has been removed and replaced with `bgp_peer_groups.ipv4_underlay_peers` to avoid upper-case variables.
 - `bgp_peer_groups.MLAG_IPv4_UNDERLAY_PEER` has been removed and replaced with `bgp_peer_groups.mlag_ipv4_underlay_peer` to avoid upper-case variables.
 - `bgp_peer_groups.EVPN_OVERLAY_PEERS` has been removed and replaced with `bgp_peer_groups.evpn_overlay_peers` to avoid upper-case variables.
+
+#### Several internal switch.* facts and avd_switch_facts have been removed
+
+As part of optimizing performance and code reuse, the amount of `switch.*` facts has been limited.
+
+The `switch.*` and `avd_switch_facts` are internal datamodels, that are used for sharing data between various stages of `eos_designs`. They **should not**
+be used in custom templates for interface descriptions or IP addressing, since they are subject to change without notice.
+
+See the porting guide for suggestions on how to leverage AVD logic in combination with Ansible inline Jinja2.
```

Comment:
Maybe add a link here

Reply:
The section is not there yet (we need that other PR with the lookup)